src/applications/vaos/sass/vaos.scss

```diff
@@ -46,3 +46,8 @@ h1:focus {
     top: 40%;
   }
 }
+
+.vaos__external-link {
+  height: 12px;
+  margin: 5px;
+}
```

Comment:
Just some formatting for the link image. Read an article that suggested that in addition to announcing that a link opens in a new tab, there should be some visual indication as well (see screenshot).

[https://medium.com/@svinkle/why-let-someone-know-when-a-link-opens-a-new-window-8699d20ed3b1](https://medium.com/@svinkle/why-let-someone-know-when-a-link-opens-a-new-window-8699d20ed3b1)